nikola/data/themes/base/templates/feeds_translations_helper.tmpl

```diff
@@ -1,91 +1,97 @@
 ## -*- coding: utf-8 -*-
 
-## Handles both feeds and translations
-<%def name="head(classification=None)">
-    % if rss_link:
+<%def name="_append_language(language)">${ " (" + language + ")" if len(translations) > 1 else "" }</%def>
```

Comment:
(FYI: I need this "the whole macro in one line" because it seems to be impossible to get rid of the generated whitespace in Mako otherwise. And if these macros generate superfluous whitespace, this screws up the generated link texts. In Jinja2, this could be done better with [whitespace control](http://jinja.pocoo.org/docs/2.9/templates/#whitespace-control).)